src/WebJobs.Script/Rpc/MessageExtensions/RpcMessageConversionExtensions.cs

```diff
@@ -88,7 +88,10 @@ public static TypedData ToRpc(this object value, ILogger logger)
 
                 foreach (var pair in request.Headers)
                 {
-                    http.Headers.Add(pair.Key.ToLowerInvariant(), pair.Value.ToString());
+                    if (!string.IsNullOrEmpty(pair.Value))
+                    {
```

Comment:
Looks like empty headers are valid: https://www.w3.org/Protocols/rfc2616/rfc2616-sec2.html#sec2.1 Do we need this change?

Reply:
This is in response to this issue: https://github.com/Azure/azure-functions-language-worker-protobuf/issues/21

But it is a breaking change for Java that does this correctly... I'll remove this until we solve the Buffer overrun problem for all languages.

Reply:
Is it possible to handle this in nodejs worker?

Reply:
No, at that point the buffer overrun has already happened and the message is corrupted :( we could remove empty headers if the language is node though..??